### PR TITLE
init

### PR DESCRIPTION
**Pull request by TerribleDev**

### CODE REVIEW

File: lib/builders/nugetPushArgBuilder.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if(!value){
+        return args;
+    }
+    if (value.forceEnglishOutput) {
+        args = args.concat(['--force-english-output']);
+    }
+    if (value.source) {
+        args = args.concat(['--source', value.source]);
+    }
+    if (value.symbolsource) {
```
Comment (neilfitzgerald) [inline lib/builders/nugetPushArgBuilder.js:12]: should this be camel cased for consistency? I.E. symbolSource

File: lib/builders/testArgBuilder.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if (value.configuration) {
+        args = args.concat(['--configuration', value.configuration]);
+    }
+    if (value.framework) {
+        args = args.concat(['--framework', value.framework]);
+    }
+    if (value.output) {
+        args = args.concat(['--output', value.output]);
+    }
+    if (value.diag) {
```
Comment (neilfitzgerald) [inline lib/builders/testArgBuilder.js:27]: Diagnostic Logging does jump out at me as what this does. I guess where it looks like a pass through its OK, but a more descriptive name might be desirable.

File: lib/builders/testArgBuilder.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+module.exports = (value) => {
+    let args = [];
+    if(!value){
```
Comment (neilfitzgerald) [inline lib/builders/testArgBuilder.js:3]: Inconsistent spacing in js files for if statement "if()" vs "if ()". There are some other formatting inconsistencies in the js files. Run a code formatter on all files.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "gulp-dotnet-cli",
+  "version": "0.0.1-alpha-1",
+  "description": "",
```
Comment (neilfitzgerald) [inline package.json:4]: "A dotnet command line interface for gulp"

**Review by neilfitzgerald — COMMENTED** [review]
Some trivial comments - code formatting & style related

File: lib/builders/testArgBuilder.js (hunk 2 above)
Comment (TerribleDev) [reply lib/builders/testArgBuilder.js:27]: is the name of the command line arg with the same name. The actual variable should be documented for users.

image

File: package.json (hunk 4 above)
Comment (TerribleDev) [reply package.json:4]: good catch!

File: lib/builders/testArgBuilder.js (hunk 3 above)
Comment (TerribleDev) [reply lib/builders/testArgBuilder.js:3]: good catch! I have jshint, but I don't believe its setup to flag these kinds of issues.

File: lib/builders/nugetPushArgBuilder.js (hunk 1 above)
Comment (TerribleDev) [reply lib/builders/nugetPushArgBuilder.js:12]: its the same name as the dotnet cli, so I think I'd prefer to keep it. However its worth debating...

**Pull request merged**